lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -30,6 +30,7 @@ def integrate!
             # TODO: refactor into Xcodeproj https://github.com/CocoaPods/Xcodeproj/issues/202
             project_is_dirty = [
               XCConfigIntegrator.integrate(target, native_targets),
+              add_pods_resources,
```

Comment:
Wondering if doing `add_pods_resources` *only* `unless native_targets_to_integrate.empty?` would be enough?